spec/models/cloud_tenant_spec.rb

```diff
@@ -9,4 +9,21 @@
 
     expect(tenant1.all_cloud_networks).to match_array([net1, net2])
   end
+
+  describe '#total_vms' do
+    let(:ems)          { FactoryGirl.create(:ems_openstack) }
+    let(:vm)           { FactoryGirl.create(:vm_openstack, :ext_management_system => ems) }
```

Comment:
can you create another vm (that is not going to come back)

So `Vm.count == 2 ; cloud_tenant.vms.count == 1`

Reply:
Working on some tests in `virtual_total_spec.rb` for that, and going to leave the other test the way it is.  They will be more encompassing, but not rely on MIQ models.

Reply:
yea, still think the addition would be useful.

Otherwise `cloud_tenant` could be dropping the whole `:tenant_id => X` portion of the `where()` and this would still pass.